### DISCUSSION TITLE
Version 1.26.2

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#441] [Announcements]
- Fix swipe gestures not working

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.26.2'>Version 1.26.2</a>.</em>

**Comment by dagimg-dot** [comment]
i really love this app, but can u add an option to change how apps are shown like vertical list without an icon.
